lib/wasix/src/syscalls/wasix/thread_spawn.rs

```diff
@@ -226,20 +235,23 @@ fn call_module<M: MemorySize>(
                         .runtime
                         .on_taint(TaintReason::UnknownWasiVersion);
                     ret = Errno::Noexec;
+                    exit_code = Some(ExitCode::Other(128 + ret as i32));
```

Comment:
what's going on here?
128 + 19, I think? Aka SIGSTOP?

That's super confusing to read.
Shouldn't we just define that as a constant and use it directly?

Reply:
Apparently, 128 + signal value is the semi-standard exit code when you receive a signal, or so the web seemed to think. No idea if this is actually a thing or not. We may as well have used a constant 1 here, but trying to follow best practices...

Reply:
But it's 128+ret, and with the offset of ret being Noexec that would amount to what's associated with SIGSTOP, so that's fine!

This is really only a readability nit.

Reply:
Let's leave it in, so people will later find and think we knew what we were doing and be amazed that we knew it.